packages/calypso-color-schemes/src/shared/color-schemes/_aquatic.scss

```diff
@@ -126,6 +126,16 @@
 	--color-sidebar-submenu-text: var(--studio-white);
 	--color-sidebar-submenu-hover-text: var(--studio-yellow-20);
 
+	/* Sidebar Menu - Nav Redesign */
+	--color-navredesign-sidebar-menu-selected-text: var(--studio-blue-90);
+
+	/* Sidebar M\Subenu - Nav Redesign */
+	--color-navredesign-sidebar-submenu-text: var(--color-sidebar-submenu-text);
+	--color-navredesign-sidebar-submenu-hover-text: var(--color-sidebar-submenu-hover-text);
+
+	/* Collapse Menu Button Fix */
```

Comment:
Why word `Fix` in the comment? What did you mean?

Reply:
On the first couple of schemes I fixed there was an existing comment for another "fix" item that I picked this wording up from. I think it was the notification bell, if I remember correctly. I then copy/pasted and tweaked that across all the schemes so the comment went to everything.

You make a good point though, this really doesn't have to say "fix", it could just be a one-line section for the Collapse button.